logsmanagement/logsmanagement_conf.h

```diff
@@ -0,0 +1,75 @@
+/** @file logsmanagement_conf.h
+ *  @brief Configuration settings for the logs management engine
+ *
+ *  @author Dimitris Pantazis
+ */
+
+#ifndef LOGSMANAGEMENT_CONF_H_
+#define LOGSMANAGEMENT_CONF_H_
+
+/* -------------------------------------------------------------------------- */
+/*                                  General                                   */
+/* -------------------------------------------------------------------------- */
+
+#define KiB * 1024
+#define MiB * 1048576
+#define GiB * 1073741824
+
+#define MS_IN_SEC 1000                              /**< Milliseconds in a second **/
```

Comment:
this is a duplication for https://github.com/netdata/netdata/blob/7b078bc3aa4eaa0781dbc55589d83ce241f1410d/libnetdata/clocks/clocks.h#L73

Reply:
Fixed!